### ISSUE TITLE
3.4.4 has been released. 

### ISSUE THREAD

**Issue by jjasghar** [jjasghar/rabbitmq#222]
Check out @RabbitMQ's Tweet: https://twitter.com/RabbitMQ/status/565801157361401857?s=09

**Comment by michaelklishin** [comment]
:+1:

**Comment by jjasghar** [comment]
It seems that CentOS hasn't released this package yet:
```
10:26:49 rabbitmq > (3.4.4) chef exec kitchen list
Instance                                     Driver        Provisioner  Last Action
default-centos-6-5-x64                       Digitalocean  ChefZero     <Not Created>
default-centos-7-0-x64                       Digitalocean  ChefZero     <Not Created>
default-debian-7-0-x64                       Digitalocean  ChefZero     <Not Created>
default-ubuntu-12-04-x64                     Digitalocean  ChefZero     <Not Created>
default-ubuntu-14-04-x64                     Digitalocean  ChefZero     <Not Created>
default-use-distro-version-centos-6-5-x64    Digitalocean  ChefZero     Created
default-use-distro-version-centos-7-0-x64    Digitalocean  ChefZero     Created
default-use-distro-version-debian-7-0-x64    Digitalocean  ChefZero     <Not Created>
default-use-distro-version-ubuntu-12-04-x64  Digitalocean  ChefZero     <Not Created>
default-use-distro-version-ubuntu-14-04-x64  Digitalocean  ChefZero     <Not Created>
cluster-centos-6-5-x64                       Digitalocean  ChefZero     <Not Created>
cluster-centos-7-0-x64                       Digitalocean  ChefZero     <Not Created>
cluster-debian-7-0-x64                       Digitalocean  ChefZero     <Not Created>
cluster-ubuntu-12-04-x64                     Digitalocean  ChefZero     <Not Created>
cluster-ubuntu-14-04-x64                     Digitalocean  ChefZero     <Not Created>
lwrps-centos-6-5-x64                         Digitalocean  ChefZero     <Not Created>
lwrps-centos-7-0-x64                         Digitalocean  ChefZero     <Not Created>
lwrps-debian-7-0-x64                         Digitalocean  ChefZero     <Not Created>
lwrps-ubuntu-12-04-x64                       Digitalocean  ChefZero     <Not Created>
lwrps-ubuntu-14-04-x64                       Digitalocean  ChefZero     <Not Created>
10:27:06 rabbitmq > (3.4.4)
```

**Comment by jjasghar** [comment]
https://github.com/jjasghar/rabbitmq/pull/223 is ready when CentOS distro pkg's are posted.

**Comment by jjasghar** [comment]
Looks like it's still not released:
```
        - install version 17.4-1.el6 of package erlang
       Recipe: rabbitmq::default


           ================================================================================
           Error executing action `install` on resource 'yum_package[rabbitmq-server]'
           ================================================================================

           Chef::Exceptions::Package
           -------------------------
           Version 3.4.4 of rabbitmq-server not found. Did you specify both version and release? (version-release, e.g. 1.84-10.fc6)

           Resource Declaration:
           ---------------------
           # In /tmp/kitchen/cache/cookbooks/rabbitmq/recipes/default.rb

           108:     package 'rabbitmq-server' do
           109:       action :install
           110:       version node['rabbitmq']['version']
           111:     end
           112:   else

           Compiled Resource:
           ------------------
           # Declared in /tmp/kitchen/cache/cookbooks/rabbitmq/recipes/default.rb:108:in `from_file'

           yum_package("rabbitmq-server") do
             action [:install]
             retries 0
             retry_delay 2
             default_guard_interpreter :default
             package_name "rabbitmq-server"
             version "3.4.4"
             timeout 900
             flush_cache {:before=>false, :after=>false}
             declared_type :package
             cookbook_name "rabbitmq"
             recipe_name "default"
           end


       Running handlers:
       [2015-02-16T17:44:18-05:00] ERROR: Running exception handlers
       Running handlers complete
       [2015-02-16T17:44:18-05:00] ERROR: Exception handlers complete
       [2015-02-16T17:44:18-05:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
       Chef Client failed. 9 resources updated in 85.756746741 seconds
       [2015-02-16T17:44:18-05:00] ERROR: yum_package[rabbitmq-server] (rabbitmq::default line 108) had an error: Chef::Exceptions::Package: Version 3.4.4 of rabbitmq-server not found. Did you specify both version and release? (version-release, e.g. 1.84-10.fc6)
       [2015-02-16T17:44:19-05:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by cmluciano** [comment]
Think this can be closed via #223
